File: CP/dmoj/rust/src/bin/ccc11s2.rs
////ccc11s2

use std::io::{self, Read};

fn main() {
    let mut input = String::new();
    io::stdin().read_to_string(&mut input).unwrap();
    let mut lines = input.lines();

    let n: usize = lines.next().unwrap().parse().unwrap();

    let student: Vec<&str> = (0..n).map(|_| lines.next().unwrap()).collect();
    let answer: Vec<&str> = (0..n).map(|_| lines.next().unwrap()).collect();

    let count = (0..n).filter(|&i| student[i] == answer[i]).count();

    println!("{}", count);
}

File: dmoj/python_cpp/ccc11s2.py
# ccc11s2

n = int(input())
student_answer = [input() for _ in range(n)]
correct_answer = [input() for _ in range(n)]

count = 0

for i in range(n):
    if student_answer[i] == correct_answer[i]:
        count += 1

print(count)